src/docs/product/releases/health/index.mdx

```diff
@@ -32,7 +32,7 @@ Number of users that started the application at least once in the last 24 hours.
 
 ### Crash
 
-The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in Sentry that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `handled:no`.
+The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in Sentry that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `handled:no`. The number of unhandled events is not expected to match the number of crashed sessions as sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) and [Sampling](/platform-redirect/?next=/configuration/sampling/).
```

Comment:
Made a small style fix and edited one word for clarity

```suggestion
The app had an explicit unhandled error or hard crash. You'll typically be able to view the corresponding issue in [sentry.io](https://sentry.io) that captures this event, and errors that did not cause the end of the application should not be included. To search for them in **Discover** or on the **Issues** page, filter by `handled:no`. The number of unhandled events is not expected to match the number of crashed sessions because sessions are not subject to [Inbound Filters](/product/data-management-settings/filtering/) and [Sampling](/platform-redirect/?next=/configuration/sampling/).
```